packages/SwingSet/src/devices/vat-admin/device-vat-admin.js

```diff
@@ -92,6 +94,7 @@ bundleID before submitting to the kernel), or (temporarily) a full bundle.
 export function buildDevice(tools, endowments) {
   const { hasBundle, getBundle, getNamedBundleID } = endowments;
   const { pushCreateVatBundleEvent, pushCreateVatIDEvent } = endowments;
+  const { pushUpgradeVatEvent } = endowments;
   const { terminate } = endowments;
   const { meterCreate, meterAddRemaining, meterSetThreshold, meterGet } =
     endowments;
```

Comment:
Any particular reason for having 5 separate destructing assignments from `endowments` instead of 1?

Reply:
Just trying to keep `prettier` from splitting it up into a gigantic vertical sprawl, which I found less readable. Not sure if my approach is the best, and I think it got worse over time (slow creep of additional endowments) but I'm increasingly annoyed at `prettier`'s enthusiasm for increasing the line count. Next time I'm in this file I'll probably look for a better approach.